cranelift/codegen/src/isa/x64/mod.rs

```diff
@@ -157,6 +157,10 @@ impl TargetIsa for X64Backend {
     fn text_section_builder(&self, num_funcs: u32) -> Box<dyn TextSectionBuilder> {
         Box::new(MachTextSectionBuilder::<inst::Inst>::new(num_funcs))
     }
+
+    fn function_alignment(&self) -> u32 {
+        16
```

Comment:
Could you leave a comment here for why this is 16? (reading this in isolation I'd naively expect it to be 1)

Reply:
Great point, added!